README.md

```diff
@@ -1,436 +1,119 @@
-## Metrilo custom integration tutorial ##
 
-_This guide is from developers for developers. If your are not a developer, please find one and come back later :)_
 
 
 
-**:warning: :warning: :warning:
-We strongly recommend doing this on Staging/Beta environment first to make sure the integration is smooth and nothing breaks down.**
+# Metrilo custom integration tutorial
 
-Metrilo provides a javascript library for integration with your custom e-commerce solution.
-The same library is used for our Magento and Woocommerce plugins so they can be a valuable source when integrating Metrilo with your website. You can find them on:
-- [Magento Plugin](https://github.com/Metrilo/Magento-Plugin)
-- [Woocommerce Plugin](https://github.com/Metrilo/WooCommerce-Plugin)
+**This guide is from developers for developers. If your are not a developer, please find one and come back later :)**
 
-This guide consists of 2 parts:
+:warning: **We strongly recommend doing this on Staging/Beta environment first to make sure the integration is smooth and nothing breaks down.**
 
-- [Front-end Metrilo integration](#front-end-metrilo-integration)
-- [Back-end Metrilo integration](#back-end-metrilo-integration)
 
+To do your custom integration with Metrilo and send data to our servers you have to use the **Metrilo API**. Its documentation is written according to the [OpenAPI Specification](https://swagger.io/docs/specification/about/). Please copy the content of this [file](metrilo_open_api_specification.yml) to the [swagger editor](https://editor.swagger.io/) for a better look. In the documentation you will find what endpoints the Metrilo API serves and what data they require.
 
-## Front-end Metrilo integration ##
+In general we can distinguish 3 types of endpoints according to their purpose. Also, this is what you have to do to complete your integration. These are endpoints for:
+* [Import of resources](#import-of-resources)
+* [Frontend integration](#frontend-integration)
+* [Backend integration](#backend-integration)
 
-_You should add the Metrilo javascript snippet on every page you want to track!_
+## Import of resources
+  Before doing any [Frontend](#frontend-integration) and [Backend](#backend-integration) integrations, you have to sync (import) your data with Metrilo. These is the **categories**, **products**, **customers** and **orders** you have in your database. Metrilo needs information about them, because these are the objects (resources) related to your customers actions.
 
-#### Installing the javascript library ####
+:warning: Keep in mind that:
+  * Import of categories and products is **crucial** because the [Frontend integration](#frontend-integration) depends on it.
+  * Import of customers and orders is necessary only if you want to sync any historical data with Metrilo.
```

Comment:
An order for an existing customer..., existing where? I am not sure what you mean by this and why it is not entirely true?

Reply:
> Import of customers and orders is necessary only if you want to sync any historical data with Metrilo.

I'm referring to this not being entirely true. What if you don't import existing (in their shop) customers and one of them (existing customers) makes an order?

Reply:
it will create a customer in Metrilo, at least this is how the processing works now. However, I am not sure if we plan to keep doing it. If not, then they will have to import customers as well. Then comes the question, what about orders that are historic, but change their statuses? We will treat them as new orders made some time ago before the integration. 
Maybe we should write import of everything is a must?

Reply:
> it will create a customer in Metrilo

If this is the case, then OK, lets leave it like that.